src/main/java/com/xceptance/xlt/mastercontroller/Main.java

```diff
@@ -75,12 +75,12 @@
 
     private static final Log log = LogFactory.getLog(Main.class);
 
-    private static BasicConsoleUI ui;
+    private BasicConsoleUI ui;
```

Comment:
Allow subclasses to access this field (w/o the need to override `setupUi`)?

Reply:
Done.